src/main/java/org/apache/mesos/hdfs/state/LiveState.java

```diff
@@ -71,25 +73,23 @@ public void updateTaskForStatus(Protos.TaskStatus status) {
       // If initializing the first NN or reconciling the first NN or bootstrapping the first NN
       // set the status to initialized
       if (status.getMessage().equals(HDFSConstants.NAME_NODE_INIT_MESSAGE)
```

Comment:
I would choose //NOPMD comments over xml.   This is a lot like choosing annotations over a bunch of xml.